src/components/map/index.tsx

```diff
@@ -142,7 +134,7 @@ class DeckGLMap extends React.Component<DeckGLPropType, DeckGLStateType> {
           }
         },
         getLineColor: [247, 105, 6, 255],
-        visible: treesVisible,
+        visible: visibleLayer === 'trees' ? true : false,
```

Comment:
The condition `visibleLayer === 'trees'` already returns a boolean. No need to use a ternary operator here.